Model/review.model.js

```diff
@@ -10,6 +10,16 @@ exports.selectReviewsById = (review_id) => {
         } else {
           return reviewOfIdNumber.rows[0];
         }
+      })
+      .then((reviewById) => {
+        const id = reviewById.review_id;
+        return db
+          .query("SELECT * FROM comments WHERE review_id = $1;", [id])
+          .then((results) => {
+            const numberOfComments = results.rows.length;
+            reviewById.comment_count = numberOfComments;
+            return reviewById;
```

Comment:
I would prefer that this is done in SQL using the COUNT() aggregate function with a join. Saves us doing two queries everytime and SQL is very good at these operations. Have a look at the notes for SQL relationships 
https://notes.northcoders.com/courses/js-back-end/sql-relationships